tests/test_slogpy_console.py

```diff
@@ -0,0 +1,16 @@
+"""Import slogpy from slog"""
+from slogpy.slog import Slog as slog  # noqa: N813
+
+"""This is a setup function"""
+def setupfunction():
+    """Not to create a file for output"""
+    slog.initialize(file_logging=False)
+
+"""First unit test"""
+def test_info_console_output(capsys, msg="This is a console test"):
+    """Passing a message to the info method and then capture the message from the console"""
```

Comment:
suggestion: using the Given-When-Then syntax will make the purpose of this test clearer ex:
Given: using slogpy
When: use info() with a string parameter
Then: the string is output to the console AND nothing is output to stderr

Reply:
I have started using this syntax. It makes the code more informative. This is a game changer. Thank you.

Reply:
do not resolve a conversation until the code is changed (in the PR) or not changing it has been discussed.